maps/torch/items/summarydocuments.dm

```diff
@@ -1,130 +1,114 @@
+	desc = "Some \"internal use only\" reports about the potential crush hazards of blast doors. It cites an incident regarding improperly conducted tests that resulted in animal cruelty charges."
 
 /obj/item/document/generic/sol/concurrentmissions
-	name = "risk assessment and officers report"
-	desc = "Some \"internal use only\" reports, containing a Risk assessment and an Officer's report. This one details concurrent shuttlecraft missions, regarding the risks and the likelihood of success. The Risk assessment states that the greatest risk comes from personnel being left behind or forgotten. The Officer's report details several key points for successful, safe missions. It notes accurate, well-communicated manifests are essential, with the Parent Vessel taking responsibility for coordination and pickups of shuttlecraft."
+	name = "risk assessment and officer's report"
+	desc = "Some \"internal use only\" reports, containing a risk assessment and an officer's report. This one details the risks and likelihood of success of running concurrent shuttlecraft missions. The risk assessment states that the greatest risk comes from personnel being left behind or forgotten. The officer's report details several key points for successful, safe missions. It notes that accurate, well-communicated manifests are essential, with the parent vessel taking responsibility for coordination and pickups of shuttlecraft."
 
 /obj/item/document/generic/sol/consecutivemissions
-	name = "risk assessment and officers report"
-	desc = "Some \"internal use only\" reports, containing a Risk assessment and an Officer's report. This one details consecutive shuttlecraft missions, regarding the risks and the likelihood of success. The Risk assessment states that the greatest risk comes from equipment misappropriation with intent to use on other missions. The Officer's report details several key points to take into account, when forming expectations of consecutive missions. It notes that the more green or disorganized a team is, the chance of added risk rises drastically after the first mission, without a suitable period to reorganize."
-
-/obj/item/document/generic/sol/holocalletiquette
-	name = "holocall etiquette"
-	desc = "Some \"internal use only\" reports, containing a small guide on proper use of holocalls from the perspective of the SEV Torch. This one details five points. First, Calls are a one-way interaction. They inform you of the basic situation, you inform them of what you can do to help. Second, Calls are two-way if they request them to be. Third, If an executive decision is required, ask the Senior on the ground. If he does not render a decision, the SEV Torch shall, but with an emphasis on being decisive. Fourth, State aloud when the call connects. It helps cut down on background chatter. Fifth, You don't need to be informed on every specific thing Exploration is doing while on Holocall."
```

Comment:
After some thinking, I decided it's best to leave this one out. There's no way to condense it to make it smaller or more easily readable, and I think making holocall etiquette prescriptive like this isn't that great of a move anyway.

Reply:
Agreed.
A shame, but I'm unable to come up with ways to condense it.